src/logger.rs

```diff
@@ -89,8 +97,13 @@ impl fmt::Display for Level {
 /// logger.log(Level::Debug, format!("feeds.len() == {}", 42));
 /// ```
 pub struct Logger {
+    /// The file to which all messages at and above `loglevel` will be written.
     logfile: Option<File>,
 
+    /// The file to which all Level::UserError messages will be written if `loglevel` is not
+    /// Level::None.
+    errorlogfile: Option<File>,
```

Comment:
When will this be used? This is a new feature Newsboat doesn't have at the moment, does it?

Shouldn't the field be called `usererrorlogfile` (or even `user_error_log_file` for increased readability, not sure what the Rust style guide says) to better reflect its purpose? Even though the name would be longer it reduces the chance of mixing things up accidentally, as the name suggests that the `Error` not the `UserError` log messages will be recorded here.

Reply:
> When will this be used? This is a new feature Newsboat doesn't have at the moment, does it?

Nah, that's good old [`error-log`](https://newsboat.org/releases/2.13/docs/newsboat.html#error-log).

> Shouldn't the field be called usererrorlogfile […] to better reflect its purpose?

The point of this PR was to move code from C++ to Rust, only making the unavoidable changes.

But since we have the boyscout rule in our coding guide, I can't really say "no" to your comment :) I'll make another commit to rename this field to `user_error_log_file`, as you suggest. Pity we can't rename the option as easily.

Thanks!

Reply:
Ah, that's that one, totally makes sense now! I didn't think of config options, I somehow assumed that it was a command line argument, but `newboat --help` didn't show me something matching. :-) So then maybe just leave it like that as it's not new contrary to my thoughts.

Reply:
Actually, no; I'd rather do this change in a separate PR, bundled with a few more minor edits I came up with while working on this PR. Are you okay with that, @der-lyse?

Reply:
> So then maybe just leave it like that as it's not new contrary to my thoughts.

No, I think current name *is* confusing and should be changed. But I'll do it in another PR.